.github/workflows/builds.yml

```
@@ -0,0 +1,119 @@
+name: Dev Releases
+
+on:
+  push:
+    branches:
+      - dev
+
+env:
+  LANG: en_US.UTF-8
+  ENVFILE: .env.dev
+  # App runtime environment
+  API_URL: ${{ secrets.DEV_API_URL }}
+  # S3
+  FOLDER: react-native-base
+  AWS_REGION: us-east-1
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  KEYS_BUCKET: ${{ secrets.AWS_S3_KEYS_BUCKET }}
+  BUILDS_BUCKET: ${{ secrets.AWS_S3_BUILDS_BUCKET }}
+  # iOS Release
+  APPLE_PROFILE: ${{ secrets.APPLE_DEV_PROFILE }}
+  APPLE_CERT: ios_distribution.cer
+  APPLE_KEY: Certificates.p12
+  APPLE_KEY_PASSWORD: ${{ secrets.APPLE_KEY_PASSWORD }}  
+  APPLE_TEAM_ID: ${{ secrets.APPLE_TEAM_ID}}
```

```suggestion
  FASTLANE_TEAM_ID: ${{ secrets.APPLE_TEAM_ID}}
```

It's later used in the Appfile and Fastfile with that name